source/manual/debian-packaging.html.md

```diff
@@ -10,6 +10,95 @@ review_in: 6 months
+
+If you are satisfied that adding a local or third party package via aptly is the
+optimal solution, there are a number of ways packages may be sourced and added,
+all carrying different advantages and disadvantages as well as security and
+maintenance implications.
+
+### Creating and maintaining packages locally
+
+- Creating local packages allows for the greatest flexibility with respect to
+  installation, default configuration and fullfilment of requirements.
+
+- Locally developed bug-fixes can be included and distributed independent of
+  distribution services.
+
+- Distribution bug-fixes and updates are not available and the package has to
+  be actively maintained by GOV.UK.
+
+- To minimise technical debt on future maintenance of packages, locally created
+  packages should only be considered if there is no reliable and secure third
+  party option available.
+
+### Mirroring packages maintained by a third party
+
+- If considering the use of a package created by a third party, the
+  trustworthiness of said party is critical.
+
+- General Debian packages explicitly labelled as suitable for Ubuntu Trusty,
+  such as Ubuntu packages provided by PostgreSQL or PPA packages by LibreOffice
+  can be considered safe.
+
+- With regard to packages provided by private contributors and/or smaller companies,
+  both the maturity of the software and trust for the source need to be evaluated.
+  Comments on maturity as well as the number of users of a package may be an
+  indicator of its suitability for use in production.
+
+- Development activity and update frequency are additional factors to consider.
+
+- When in doubt about the trustworthiness or stability of a third party package,
+  this option should be disregarded in favour of other solutions not
+  compromising on reliability and security.
+
+### Mirroring or importing .deb packages created for other distributions (e.g. Debian, Linux Mint, etc)
+
+- If great care is taken, it may be possible to make use of a software version
+  which has been packaged for another Linux distribution, e.g. Debian.
+
+- Despite Ubuntu being based on Debian experimental, no Debian repository should ever
+  be directly mirrored and made available through aptly to a Ubuntu system _ever_.
+  There will be severe complications including failing or erroneous package upgrades,
+  broken dependencies and likely complete system failure.
+
+- The safest way to attempt to use a .deb package of another distro is to include its
+  source repository and compile the package yourself. After adding e.g. the Debian source
+  repository `deb-src http://deb.debian.org/debian/ experimental main contrib non-free`
+  and executing `apt-get update`, compilation of the desired software can be
+  attempted by invoking:
+
+  ```
+  apt-get build-dep <package> # For build dependencies
+  apt-get source --compile <package> # To download source and compile package
+  ```
```

Comment:
I think this section has got a little too technical for these bullet points and may belong as part of a wider description of how to use debian packages?